ocaml/xenopsd/lib/xenops_utils.ml

```diff
@@ -115,27 +115,17 @@ let ignore_bool (_ : bool) = ()
 let ignore_int (_ : int) = ()
 
 (* Recode an incoming string as valid UTF-8 *)
+let utf8_fold buf _ next =
```

Comment:
This is not a good name - this simply adds to the buffer - so it should not contain `fold`.